### DISCUSSION TITLE
[Feature Request] Input field component for manual color value entry

### DISCUSSION THREAD

**Discussion by advename** [jaames/iro.js#178] [Feature Requests]
It would be nice to have some kind of additional input field to manually insert an rgb or hex value.

**Comment by jaames** [comment]
I think this is pretty easy to implement yourself with a html `<input>` element, so I'm gonna wait on this for now. If anyone else wants this feature, please leave a 👍 on advename's message. Thanks!

**Comment by jcrawford** [comment]
I have been searching for a way to do this even with implementing my own input box, however I do not see any method that will allow me to update the actively selected color in the wheel.

**Comment by jcrawford** [comment]
I created #150 as I didn't see this ticket at first.
